specification/compute/resource-manager/readme.md

```diff
@@ -209,11 +209,35 @@ directive:
       - ACS service is deprecated so a PATCH endpoint won't be implemented
 
 ```
+### Tag: package-2020-12-01-preview
+
+These settings apply only when `--tag=package-2020-12-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2020-12-01-preview'
+input-file:
+- Microsoft.Compute/stable/2020-12-01/compute.json
+- Microsoft.Compute/stable/2020-12-01/runCommands.json
+- Microsoft.Compute/stable/2019-04-01/skus.json
+- Microsoft.Compute/stable/2020-09-30/disk.json
+- Microsoft.Compute/stable/2019-12-01/gallery.json
+- Microsoft.Compute/preview/2020-10-01-preview/cloudService.json
```

Comment:
I suggest that we introduce another tag which only contains the stable swaggers or remove the preview swagger from this tag. For a RP like compute, SDK users might have concerns to use a preview package.

Reply:
@ArcturusZhang 
Our current plan is to **not** mark this Rest API release as '-preview' but leave cloudservices.json in the release in the preview folder. 

The CloudService is a new service added in the previous release in public preview.
There is almost always a feature in public preview present in the Compute folder because there are so many service teams that contribute to this swagger. Therefore we cannot flag the SDK packages with -preview since all packages will have -preview. 
We cannot remove the CloudServices swagger because that would break our .Net SDK test code, and remove functionality we had released. 

We are working on a better release plan for preview features, but that will be implemented after this 2020-12-01 release. For now we will proceed with this release without -preview.